src/app/wallet/wallet.component.ts

```diff
@@ -71,7 +72,10 @@ export class WalletComponent implements OnInit {
     getTransactionHistory() {
         SpinnerUtil.showSpinner();
         this.walletService.getTransactionHistory().subscribe((res: TransactionList) => {
-            this.transactionHistory = res.transactions;
+            this.transactionHistory = res.transactions
+                .sort((a,b) => {
+                    return new Date(b.date).getTime() - new Date(a.date).getTime()
+                })
```

Comment:
This can be oneliner
```suggestion
                .sort((a,b) => new Date(b.date).getTime() - new Date(a.date).getTime())
```